protocol/json-schemas/tbdex-message-schema.json

```diff
@@ -0,0 +1,276 @@
+                },
+                "payinInstruments": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/paymentInstrument"
+                    }
+                },
+                "payoutInstruments": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/paymentInstrument"
+                    }
+                }
+            },
+            "required": [
+                "description",
+                "pair",
+                "unitPrice",
+                "min",
+                "max",
+                "presentationRequestJwt",
+                "payinInstruments",
+                "payoutInstruments"
+            ]
+        },
+        "rfq": {
+            "type": "object",
+            "properties": {
+                "pair": {
+                    "type": "string"
+                },
+                "amount": {
+                    "type": "string"
+                },
+                "verifiablePresentationJwt": {
+                    "type": "string"
+                },
+                "payinInstrument": {
+                    "$ref": "#/definitions/paymentInstrument"
+                },
+                "payoutInstrument": {
+                    "$ref": "#/definitions/paymentInstrument"
+                }
+            },
+            "required": [
+                "pair",
+                "amount",
+                "verifiablePresentationJwt",
+                "payinInstrument",
+                "payoutInstrument"
+            ]
+        },
+        "quote": {
+            "type": "object",
+            "properties": {
+                "expiryTime": {
+                    "type": "number"
+                },
+                "totalFee": {
+                    "type": "string"
+                },
+                "amount": {
+                    "type": "string"
+                },
+                "paymentPresentationRequestJwt": {
+                    "type": "string"
+                },
+                "paymentInstructions": {
+                    "$ref": "#/definitions/paymentInstructions"
+                }
+            },
+            "required": [
+                "expiryTime",
+                "totalFee",
+                "amount",
+                "paymentPresentationRequestJwt",
+                "paymentInstructions"
+            ]
+        },
+        "order": {
+            "type": "object",
+            "properties": {
+                "paymentVerifiablePresentationJwt": {
+                    "type": "string"
+                },
+                "required": [
+                    "paymentVerifiablePresentationJwt"
+                ]
+            }
+        },
+        "orderStatus": {
+            "type": "object",
+            "properties": {
+                "orderStatus": {
+                    "type": {
+                        "enum": [
+                            "PENDING"
```

Comment:
we could add `INPROGRESS` and `COMPLETED` ?

Reply:
what's the difference between `PENDING` and `IN_PROGRESS`?

Reply:
nothing! they're the same in my mind. i was saying that we should probably come up with all possible order states. so it could be something along the lines of`PENDING`, `COMPLETED`, `ERROR` ?

Reply:
added a couple but I think we'll have to revise as we go. For ex., can fail for pay in or pay out side, but do we want to distinguish that granularity here? and other questions like that.